x-pack/plugins/security_solution/common/endpoint/service/response_actions/utils.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ConsoleResponseActionCommands, ResponseActionsApiCommandNames } from './constants';
+import { RESPONSE_CONSOLE_ACTION_COMMANDS_TO_REQUIRED_AUTHZ } from './constants';
+import type { EndpointPrivileges } from '../../types';
+
+export const getRbacControl = ({
+  commandName,
+  privileges,
+}: {
+  commandName: ConsoleResponseActionCommands;
+  privileges: EndpointPrivileges;
+}): boolean => {
+  return Boolean(privileges[RESPONSE_CONSOLE_ACTION_COMMANDS_TO_REQUIRED_AUTHZ[commandName]]);
+};
+
+/**
+ * map actual command to ui command
+ * unisolate -> release
+ * running-processes -> processes
+ */
+export const getUiCommand = (
+  command: ResponseActionsApiCommandNames
+): ConsoleResponseActionCommands => {
+  if (command === 'unisolate') {
+    return 'release';
+  } else if (command === 'running-processes') {
+    return 'processes';
+  } else {
+    return command;
+  }
+};
```

Comment:
we already have a `const` that has this map here: 

https://github.com/elastic/kibana/blob/85245e13e6aba610a5a336b3528888c712f24d71/x-pack/plugins/security_solution/common/endpoint/service/response_actions/constants.ts#L95-L106

Can we just use it?

Reply:
Also - do we need this function on the server side (is it shared with server-side code)? if not, then I would move it to be under our `public/management` folder instead.

Reply:
updated 526c540341de045581c9e0039297f0ec93ccafcc